core/blockchain.go

```diff
@@ -258,12 +258,27 @@ type BlockChain struct {
 	forker     *ForkChoice
 	vmConfig   vm.Config
 	logger     *tracing.Hooks
+
+	// true if witness trie nodes will be prefetched during block execution. The prefetcher will also terminate
+	// synchronously in this case.
+	collectWitness bool
```

Comment:
Bonus point if we can get this to be part of the vmconfig (thinking long term that if we're going to replace it with some xcross referencing validation config, that would also go there I guess?)